src/spring.js

```diff
@@ -230,14 +230,15 @@ $.Spring.prototype = {
                     ( this.target.time - this.start.time )
                 );
 
-        var oldValue = this.current.value;
+        // var oldValue = this.current.value;
         if (this._exponential) {
             this.current.value = Math.exp(currentValue);
         } else {
             this.current.value = currentValue;
         }
 
-        return oldValue !== this.current.value;
+        return currentValue !== targetValue;
```

Comment:
Can you explain this fix?

Reply:
As I was testing new features - I believe it was related to the process of separating the world updates vs drawing operations - there were times when a spring was being "updated" twice in a short enough time that the spring value wasn't changed by consecutive calls. This meant that the spring returned `false` for being updated (even though it was not at the target value), and the 'return false' was used downstream to indicate that animating was not ongoing. This stopped the calls to keep calculating animations prematurely.

I'd be up for updating the documentation to something like "returns true if the spring needs further updates to reach the target, or false if it is already at the target" or similar wording (if you agree).

Reply:
Cool. Seems like a good improvement. And yeah, just a little update to the documentation would be good. You can also kill the commented code.